### PR TITLE
Login, Forgot Password, Set New Password - UI pages done

### PR DESCRIPTION
**Pull request by muralidharan92**
Hi @rakeshar3796 please review and merge

### CODE REVIEW

**Comment by rakeshar3796** [comment]
ok @muralidharan92

File: web/src/forgotpassword/forgotpassword.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+.forgotpassword {
+  height: 80vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.forgotpassword h1 {
+  font-size: 22px;
```
Comment (rakeshar3796) [inline web/src/forgotpassword/forgotpassword.css:9]: 2.2rem

File: web/src/forgotpassword/forgotpassword.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  font-weight: bold;
+}
+
+.forgotpassword p {
+  font-size: 13px;
```
Comment (rakeshar3796) [inline web/src/forgotpassword/forgotpassword.css:14]: 1.3rem

File: web/src/forgotpassword/forgotpassword.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+.forgotpassword form {
+  width: 400px;
```
Comment (rakeshar3796) [inline web/src/forgotpassword/forgotpassword.css:18]: 40rem
use rem values

File: web/src/forgotpassword/index.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import { Form, Input, Button } from "antd";
+import "./forgotpassword.css";
+import { Link } from "react-router-dom";
+
+export function ForgotPassword() {
+  const [email, setEmail] = useState("");
+
+  const onFinish = async (values: { email: String }) => {
```
Comment (rakeshar3796) [inline web/src/forgotpassword/index.tsx:9]: remove this method if not used

File: web/src/forgotpassword/index.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    console.log(values);
+  };
+  return (
+    <div className="forgotpassword">
+      <Form layout="vertical" onFinish={onFinish}>
+        <h1>Forgot password</h1>
+        {/* <hr /> */}
```
Comment (rakeshar3796) [inline web/src/forgotpassword/index.tsx:16]: remove comments

File: web/src/forgotpassword/index.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import { Form, Input, Button } from "antd";
+import "./forgotpassword.css";
```
Comment (rakeshar3796) [inline web/src/forgotpassword/index.tsx:3]: use module css

File: web/src/forgotpassword/index.tsx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+import { Link } from "react-router-dom";
+
+export function ForgotPassword() {
+  const [email, setEmail] = useState("");
+
+  const onFinish = async (values: { email: String }) => {
+    console.log(values);
+  };
+  return (
+    <div className="forgotpassword">
```
Comment (rakeshar3796) [inline web/src/forgotpassword/index.tsx:13]: use styles.forgotpassword

File: web/src/forgotpassword/index.tsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+        <Form.Item name="email">
+          <Input
+            autoComplete="off"
+            placeholder="Email"
+            onChange={(e) => setEmail(e.target.value)}
+          />
+        </Form.Item>
+        <div className="footer">
+          <Link to="/login">Go back to sign in</Link>
+          <Button type="primary" htmlType="submit" disabled={!email}>
```
Comment (rakeshar3796) [inline web/src/forgotpassword/index.tsx:29]: why htmlType

File: web/src/login/index.tsx [hunk 9]
```diff
@@ -1,29 +1,60 @@
-import React from "react";
-import { Input, Button } from "antd";
-
-import styles from "./login.module.css";
-
-export function LoginPage() {
-  function onSubmitHandler() {}
+import React, { useState } from "react";
+import { Form, Input, Button, message } from "antd";
+import "./login.css";
```
Comment (rakeshar3796) [inline web/src/login/index.tsx:3]: same here

File: web/src/route.tsx [hunk 10]
```diff
@@ -0,0 +1,82 @@
+import React from "react";
```
Comment (rakeshar3796) [inline web/src/route.tsx:1]: use lazily

File: web/src/setpassword/index.tsx [hunk 11]
```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import { Form, Input, Button } from "antd";
+import "./setpassword.css";
+import Circle from "./icons/circle.svg";
+import CirclePass from "./icons/circle.pass.svg";
+
+export function SetPassword() {
+  const VALIDATE_AT_LEAST_EIGHT_CHARACTER_REGEX = /^(?=\S{8,}$).*$/;
```
Comment (rakeshar3796) [inline web/src/setpassword/index.tsx:8]: move constant outside component

File: web/src/forgotpassword/forgotpassword.css (hunk 2 above)
Comment (muralidharan92) [reply web/src/forgotpassword/forgotpassword.css:14]: Changes done

File: web/src/forgotpassword/forgotpassword.css (hunk 1 above)
Comment (muralidharan92) [reply web/src/forgotpassword/forgotpassword.css:9]: Changes done

File: web/src/forgotpassword/forgotpassword.css (hunk 3 above)
Comment (muralidharan92) [reply web/src/forgotpassword/forgotpassword.css:18]: Changes done

File: web/src/forgotpassword/index.tsx (hunk 5 above)
Comment (muralidharan92) [reply web/src/forgotpassword/index.tsx:16]: Changes done

File: web/src/forgotpassword/index.tsx (hunk 6 above)
Comment (muralidharan92) [reply web/src/forgotpassword/index.tsx:3]: Changes done

File: web/src/forgotpassword/index.tsx (hunk 7 above)
Comment (muralidharan92) [reply web/src/forgotpassword/index.tsx:13]: Changes done

File: web/src/forgotpassword/index.tsx (hunk 8 above)
Comment (muralidharan92) [reply web/src/forgotpassword/index.tsx:29]: Changes done

File: web/src/login/index.tsx (hunk 9 above)
Comment (muralidharan92) [reply web/src/login/index.tsx:3]: Changes done

File: web/src/route.tsx (hunk 10 above)
Comment (muralidharan92) [reply web/src/route.tsx:1]: Changes Done

File: web/src/setpassword/index.tsx (hunk 11 above)
Comment (muralidharan92) [reply web/src/setpassword/index.tsx:8]: Changes done

File: web/src/forgotpassword/index.tsx (hunk 4 above)
Comment (muralidharan92) [reply web/src/forgotpassword/index.tsx]: Changes done

**Review by rakeshar3796 — APPROVED** [review]

**Pull request merged**